includes/Modules/Thank_With_Google/Supporter_Wall_Widget.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Thank_With_Google\Supporter_Wall_Widget
+ *
+ * @package   Google\Site_Kit\Modules\Thank_With_Google
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Thank_With_Google;
+
+use WP_Widget;
+
+/**
+ * The supporter wall widget for Thank with Google.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Supporter_Wall_Widget extends WP_Widget {
+
+	const WIDGET_ID = 'googlesitekit-twg-supporter-wall';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function __construct() {
+		parent::__construct(
+			self::WIDGET_ID,
+			sprintf( 'Thank with Google: %s', __( 'Supporter Wall', 'google-site-kit' ) )
```

Comment:
I am not sure about this... As far as I know product names are not translated since they are "international" names. For example, take a look at https://www.thinkwithgoogle.com/ - you can switch between different languages and see that "Think with Google" renames untranslated.

Reply:
From what I can see we are translating the service names, for example -

https://github.com/google/site-kit-wp/blob/cc86db8f12665a5badf9a5bbbe48d20cce087d8b/includes/Modules/Thank_With_Google.php#L117

This can be also seen for other modules and in JS too...

Reply:
Fair point, I'll update 👍

Reply:
Updated, should be good now.